converter/converters/pod/affinity.go

```diff
@@ -0,0 +1,398 @@
+package pod
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/golang/glog"
+	"github.com/koki/short/types"
+)
+
+func RevertAffinities(a []types.Affinity) (*v1.Affinity, error) {
+	node, pod, antiPod, err := splitAffinities(a)
+	if err != nil {
+		return nil, err
+	}
+
+	n, err := nodeAffinity(node)
+	if err != nil {
+		return nil, err
+	}
+
+	p, err := podAffinity(pod)
```

Comment:
podAffinities, err := revertPodAffinity(podAffinities)

Reply:
done. i left out the "affinities" part since it's a long word, and it's clear in context that `kubePod`, `kubeAntiPod`, and `kubeNode` refer to affinities.